www/js/coolpoints-util.js

```diff
@@ -24,9 +24,17 @@ function initPointsForSelf(pts) {
   const pointsEl = $("#coinAmt");
   pointsEl.text(`${pts}`);
 
-  $("#coinWrapper").addClass("cpFadeIn");
 }
-
+/**
+ * Kamal - moved the button loading out of "initPointsForSelf()"" since it is called on page load.
+ * If we wanted to put any other animations on the button for points gained or earned we'd need the previous
+ * animation class removed i.e., "cpFadeIn".
+ * 9-13-24: Calls updateAnimation now as the previous code was redundant.
```

Comment:
oh and, generally, no need to leave time stamped comments like this. git commit messages and history do a better job telling this "story" anyway